src/httpserver.cpp

```diff
@@ -618,7 +623,7 @@ CService HTTPRequest::GetPeer()
         const char* address = "";
         uint16_t port = 0;
         evhttp_connection_get_peer(con, (char**)&address, &port);
-        peer = CService(address, port);
+        peer = LookupNumeric(address, port);
```

Comment:
I like this interface change, seems a lot cleaner. A lot of this code feels like it's making the interface less ergonomic, to allow for extensibility, but this just looks like a pure quality-of-life improvement.